types/plotly.js/lib/locales/am.d.ts

```diff
@@ -0,0 +1,3 @@
+import { Locale } from '../core';
+
+export = Locale;
```

Comment:
This needs to be
```ts
const locale: Locale;
export = locale;
```
Since it exports a value of type Locale rather than the type Locale

Reply:
This does not work for me. Only this:

`import { Locale } from '../core';

const locale: Locale;

export { Locale };`